src/routes/safe/components/Apps/hooks/useAppList.ts

```diff
@@ -4,27 +4,42 @@ import { APPS_STORAGE_KEY, getAppInfoFromUrl, getAppsList, getEmptySafeApp } fro
 import { AppData } from '../api/fetchSafeAppsList'
 import { SafeApp, StoredSafeApp, SAFE_APP_FETCH_STATUS } from '../types'
 import { getNetworkId } from 'src/config'
+import enqueueSnackbar from 'src/logic/notifications/store/actions/enqueueSnackbar'
+import { NOTIFICATIONS } from 'src/logic/notifications'
+import { useDispatch } from 'react-redux'
 
 type UseAppListReturnType = {
   appList: SafeApp[]
   removeApp: (appUrl: string) => void
   staticAppsList: AppData[]
+  isLoading: boolean
 }
 
-const useAppList = (): UseAppListReturnType => {
+const useAppList = (showError: boolean): UseAppListReturnType => {
   const [appList, setAppList] = useState<SafeApp[]>([])
   const [staticAppsList, setStaticAppsList] = useState<AppData[]>([])
```

Comment:
Didn't we decide to kill the staticAppsList?

Reply:
You're right, i'm renaming the variables and deleting the static apps array in utils